test/powershell/Modules/Microsoft.PowerShell.Utility/Send-MailMessage.Tests.ps1

```diff
@@ -61,45 +64,91 @@ Describe "Send-MailMessage" -Tags CI, RequireSudoOnUnix {
+            }
+        }
+        @{
+            Name = "with multiple Cc"
+            InputObject = @{
+                From = "user01@example.com"
+                To = "user02@example.com"
+                Cc = "user03@example.com","user04@example.com"
+                Subject = "Subject $(Get-Date)"
+                Body = "Body $(Get-Date)"
+                SmtpServer = "127.0.0.1"
+            }
+        }
+        @{
+            Name = "with multiple Bcc"
+            InputObject = @{
+                From = "user01@example.com"
+                To = "user02@example.com"
+                Bcc = "user03@example.com","user04@example.com"
+                Subject = "Subject $(Get-Date)"
+                Body = "Body $(Get-Date)"
+                SmtpServer = "127.0.0.1"
+            }
+        }
         @{
             Name = "with No Subject"
             InputObject = @{
                 From = "user01@example.com"
                 To = "user02@example.com"
-                ReplyTo = "noreply@example.com"
                 Body = "Body $(Get-Date)"
                 SmtpServer = "127.0.0.1"
             }
         }
     )
 
-    It "Can send mail message using named parameters <Name>" -TestCases $testCases {
-        param($InputObject)
-
+    It "Shows obsolete message for cmdlet" {
         $server | Should -Not -Be $null
 
         $powershell = [PowerShell]::Create()
 
-        $null = $powershell.AddCommand("Send-MailMessage").AddParameters($InputObject).AddParameter("ErrorAction","SilentlyContinue")
+        $null = $powershell.AddCommand("Send-MailMessage").AddParameters($testCases[0].InputObject).AddParameter("ErrorAction","SilentlyContinue")
 
         $powershell.Invoke()
 
         $warnings = $powershell.Streams.Warning
 
         $warnings.count | Should -BeGreaterThan 0
-        $warnings[0].ToString() | Should  -BeLike  "The command 'Send-MailMessage' is obsolete. *"
+        $warnings[0].ToString() | Should -BeLike "The command 'Send-MailMessage' is obsolete. *"
+    }
+
+    It "Can send mail message using named parameters <Name>" -TestCases $testCases {
+        param($InputObject)
+
+        $server | Should -Not -Be $null
+
+        Send-MailMessage @InputObject -ErrorAction SilentlyContinue
 
         $mail = Read-Mail
 
         $mail.FromAddress | Should -BeExactly $InputObject.From
-        $mail.ToAddresses | Should -BeExactly $InputObject.To
+        $mail.ToAddresses | Should -BeIn ([array]$InputObject.To + $InputObject.Cc + $InputObject.Bcc)
 
         $mail.Headers["From"] | Should -BeExactly $InputObject.From
-        $mail.Headers["To"] | Should -BeExactly $InputObject.To
-        $mail.Headers["Reply-To"] | Should -BeExactly $InputObject.ReplyTo
-        If ($InputObject.Subject -ne $null) {
-            $mail.Headers["Subject"] | Should -BeExactly $InputObject.Subject
+        $mail.Headers["To"].Split(", ") | Should -BeExactly $InputObject.To
+        If($mail.Headers["Cc"])
```

Comment:
I think it should be check for `$InputObject`
```suggestion
        If($InputObject.Headers["Cc"])
```
Below too.

Reply:
Very good catch @iSazonov! Thanks a lot. Though the change is `$InputObject.Cc`. This also shows that `Bcc` can't be tested against the headers, since it never occurs in the headers. Which makes sense, since nobody should see that information except the mail server 🤦‍♂️.